packages/back-end/src/api/openapi/payload-schemas/PutExperimentPayload.yaml

```diff
@@ -0,0 +1,159 @@
+type: object
+required:
+  - id
+  - name
+  - variations
+  - owner
+  - datasourceId
+  - trackingKey
+  - exposureQueryId
+properties:
+  datasourceId:
+    type: string
+    description:  ID for the [DataSource](#tag/DataSource_model)
+  exposureQueryId:
+    type: string
+  trackingKey:
+    type: string
+  id:
+    type: string
+    description: ID for the experiment
+  dateCreated:
+    type: string
+    format: date-time
+  dateUpdated:
+    type: string
+    format: date-time
```

Comment:
Both `dateCreated` and `dateUpdated` are not used.  They are immediately overwritten when saving to be the current date.  So we can remove them here.

Reply:
Makes sense, will fix accordingly.